Makefile

```diff
@@ -44,43 +44,43 @@ go-gpt4all-j/libgptj.a-generic: go-gpt4all-j
 	$(MAKE) -C go-gpt4all-j generic-libgptj.a
 
 # CEREBRAS GPT
-go-gpt2.cpp:
```

Comment:
I wonder, this PR is renaming go-gpt2.cpp to go-gpt2, is that for consistency, or somehow Mac didn't liked it?

Reply:
consistancy